README.md

```diff
@@ -11,11 +11,16 @@ The easiest way to start Chronicle is by using our supplied Dockerfile.
 First you need to setup the correct permissions for the node's database:
 
 ```sh
-mkdir docker/alphanet
-
+mkdir docker/hornet_data
```

Comment:
should we really bind ourselves to `hornet`? What about `node_data`?

Reply:
This is strongly tied to `docker/docker-compose.hornet.yml`, we will have to change documentation for Bee anyways.